framework/include/utils/DelimitedFileReader.h

```diff
@@ -51,60 +59,128 @@ class DelimitedFileReader
    */
   void read();
 
+  ///@{
   /**
-   * Return the column names.
+   * Toggles for file format items.
    */
-  const std::vector<std::string> & getColumnNames() const;
+  void setIgnoreEmptyLines(bool value) { _ignore_empty_lines = value; }
+  void setFormat(const std::string & value) { _format = value; }
```

Comment:
Should this really be a std::string? You might consider a class enum. Also, you might want to add a way to query the currently set value while you are in here.

Reply:
I can make this an class Enum.